cl/sentinel/gossip.go

```diff
@@ -292,6 +317,169 @@ func (s *Sentinel) defaultVoluntaryExitTopicParams() *pubsub.TopicScoreParams {
 	}
 }
 
+func (s *Sentinel) defaultSyncSubnetTopicParams(activeValidators uint64) *pubsub.TopicScoreParams {
```

Comment:
hmm...
i should read libp2p to get more understand on it, right?

Reply:
I copied it from Prysm. but if you want to learn it https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.0.md